INFO 2025-09-30 10:40:45,322 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 10:40:57,561 basehttp "GET /profile/ HTTP/1.1" 200 19577
INFO 2025-09-30 10:42:36,971 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:42:37,629 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:42:37,892 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:42:38,137 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:42:40,993 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:42:41,199 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:42:41,377 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:42:41,561 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:42:41,721 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:43:00,169 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:43:00,415 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:43:00,609 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:43:00,800 basehttp "GET /profile/ HTTP/1.1" 200 19579
ERROR 2025-09-30 10:45:56,454 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 489, in parse
    raise self.error(token, e)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 609, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 710, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '% user.first_name %' from '% user.first_name %'
ERROR 2025-09-30 10:45:56,455 basehttp "GET /profile/ HTTP/1.1" 500 160751
ERROR 2025-09-30 10:45:57,260 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 489, in parse
    raise self.error(token, e)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 609, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 710, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '% user.first_name %' from '% user.first_name %'
ERROR 2025-09-30 10:45:57,260 basehttp "GET /profile/ HTTP/1.1" 500 160751
ERROR 2025-09-30 10:45:58,180 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 489, in parse
    raise self.error(token, e)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 609, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 710, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '% user.first_name %' from '% user.first_name %'
ERROR 2025-09-30 10:45:58,181 basehttp "GET /profile/ HTTP/1.1" 500 160751
INFO 2025-09-30 10:46:07,232 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:46:07,945 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:46:08,193 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:46:08,439 basehttp "GET /profile/ HTTP/1.1" 200 19579
INFO 2025-09-30 10:47:07,882 basehttp "GET /profile/ HTTP/1.1" 200 19586
INFO 2025-09-30 10:47:08,416 basehttp "GET /profile/ HTTP/1.1" 200 19586
INFO 2025-09-30 10:47:08,614 basehttp "GET /profile/ HTTP/1.1" 200 19586
INFO 2025-09-30 10:47:08,830 basehttp "GET /profile/ HTTP/1.1" 200 19586
INFO 2025-09-30 10:47:43,728 basehttp "GET /profile/ HTTP/1.1" 200 19587
INFO 2025-09-30 10:47:44,161 basehttp "GET /profile/ HTTP/1.1" 200 19587
INFO 2025-09-30 10:47:44,343 basehttp "GET /profile/ HTTP/1.1" 200 19587
INFO 2025-09-30 10:48:13,528 basehttp "GET /profile/ HTTP/1.1" 200 19587
INFO 2025-09-30 10:48:13,743 basehttp "GET /profile/ HTTP/1.1" 200 19587
INFO 2025-09-30 10:48:13,936 basehttp "GET /profile/ HTTP/1.1" 200 19587
INFO 2025-09-30 10:48:16,378 basehttp "GET /profile/ HTTP/1.1" 200 19587
INFO 2025-09-30 10:48:54,639 basehttp "GET /profile/ HTTP/1.1" 200 19589
INFO 2025-09-30 10:48:55,408 basehttp "GET /profile/ HTTP/1.1" 200 19589
INFO 2025-09-30 10:48:55,617 basehttp - Broken pipe from ('127.0.0.1', 37468)
INFO 2025-09-30 10:48:55,832 basehttp "GET /profile/ HTTP/1.1" 200 19589
INFO 2025-09-30 10:49:09,288 basehttp "GET /profile/ HTTP/1.1" 200 19592
INFO 2025-09-30 10:49:09,726 basehttp "GET /profile/ HTTP/1.1" 200 19592
INFO 2025-09-30 10:49:09,942 basehttp "GET /profile/ HTTP/1.1" 200 19592
INFO 2025-09-30 10:49:10,167 basehttp "GET /profile/ HTTP/1.1" 200 19592
INFO 2025-09-30 10:49:45,984 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:50:09,501 basehttp "GET /profile/ HTTP/1.1" 200 18239
INFO 2025-09-30 10:50:10,028 basehttp "GET /profile/ HTTP/1.1" 200 18239
INFO 2025-09-30 10:50:10,268 basehttp "GET /profile/ HTTP/1.1" 200 18239
INFO 2025-09-30 10:50:10,508 basehttp "GET /profile/ HTTP/1.1" 200 18239
INFO 2025-09-30 10:50:57,199 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:50:57,712 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:50:57,936 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:50:58,109 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:50:58,301 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:51:25,479 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:51:26,320 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:51:26,792 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:51:27,007 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:51:37,094 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:51:37,710 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:51:38,046 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:51:38,271 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:51:38,446 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:52:36,950 basehttp "GET /profile/ HTTP/1.1" 200 19585
WARNING 2025-09-30 10:53:49,407 log Forbidden (CSRF token missing.): /profile/
WARNING 2025-09-30 10:53:49,408 basehttp "POST /profile/ HTTP/1.1" 403 2491
INFO 2025-09-30 10:53:52,188 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:53:52,855 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:53:53,070 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:53:53,272 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:53:53,438 basehttp "GET /profile/ HTTP/1.1" 200 19585
INFO 2025-09-30 10:55:57,392 basehttp "GET /profile/ HTTP/1.1" 200 19596
INFO 2025-09-30 10:55:57,869 basehttp "GET /profile/ HTTP/1.1" 200 19596
INFO 2025-09-30 10:55:58,108 basehttp "GET /profile/ HTTP/1.1" 200 19596
INFO 2025-09-30 10:55:58,316 basehttp "GET /profile/ HTTP/1.1" 200 19596
INFO 2025-09-30 10:59:32,295 basehttp "GET /profile/ HTTP/1.1" 200 19549
INFO 2025-09-30 10:59:32,909 basehttp "GET /profile/ HTTP/1.1" 200 19549
INFO 2025-09-30 10:59:33,127 basehttp "GET /profile/ HTTP/1.1" 200 19549
INFO 2025-09-30 10:59:33,314 basehttp "GET /profile/ HTTP/1.1" 200 19549
INFO 2025-09-30 10:59:41,046 basehttp "GET /profile/ HTTP/1.1" 200 19549
INFO 2025-09-30 11:00:05,578 basehttp "GET /profile/ HTTP/1.1" 200 0
INFO 2025-09-30 11:00:06,153 basehttp "GET /profile/ HTTP/1.1" 200 0
INFO 2025-09-30 11:00:24,621 basehttp "GET /profile/ HTTP/1.1" 200 11246
INFO 2025-09-30 11:01:13,943 basehttp "GET /profile/ HTTP/1.1" 200 11237
INFO 2025-09-30 11:01:14,477 basehttp "GET /profile/ HTTP/1.1" 200 11237
INFO 2025-09-30 11:01:14,645 basehttp "GET /profile/ HTTP/1.1" 200 11237
INFO 2025-09-30 11:01:35,597 basehttp "POST /profile/ HTTP/1.1" 302 0
INFO 2025-09-30 11:01:35,601 basehttp "GET /profile HTTP/1.1" 301 0
INFO 2025-09-30 11:01:35,605 basehttp "GET /profile/ HTTP/1.1" 200 11871
INFO 2025-09-30 11:02:05,723 basehttp "POST /profile/ HTTP/1.1" 200 11642
INFO 2025-09-30 11:02:12,976 basehttp "POST /profile/ HTTP/1.1" 200 11642
INFO 2025-09-30 11:02:14,601 basehttp "POST /profile/ HTTP/1.1" 200 11642
INFO 2025-09-30 11:02:16,506 basehttp "GET /profile/ HTTP/1.1" 200 11237
INFO 2025-09-30 11:02:38,696 basehttp "POST /profile/ HTTP/1.1" 302 0
INFO 2025-09-30 11:02:38,701 basehttp "GET /profile HTTP/1.1" 301 0
INFO 2025-09-30 11:02:38,705 basehttp "GET /profile/ HTTP/1.1" 200 11877
INFO 2025-09-30 11:03:21,657 basehttp "POST /profile/ HTTP/1.1" 302 0
INFO 2025-09-30 11:03:21,661 basehttp "GET /profile HTTP/1.1" 301 0
INFO 2025-09-30 11:03:21,664 basehttp "GET /profile/ HTTP/1.1" 200 11871
INFO 2025-09-30 11:04:25,152 basehttp "GET /profile/ HTTP/1.1" 200 10941
INFO 2025-09-30 11:04:25,544 basehttp "GET /profile/ HTTP/1.1" 200 10941
INFO 2025-09-30 11:04:25,749 basehttp "GET /profile/ HTTP/1.1" 200 10941
INFO 2025-09-30 11:04:25,927 basehttp "GET /profile/ HTTP/1.1" 200 10941
ERROR 2025-09-30 11:10:38,376 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'endif'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 231: 'endif', expected 'empty' or 'endfor'. Did you forget to register or load this tag?
ERROR 2025-09-30 11:10:38,377 basehttp "GET /profile/ HTTP/1.1" 500 205796
ERROR 2025-09-30 11:10:39,152 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'endif'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 231: 'endif', expected 'empty' or 'endfor'. Did you forget to register or load this tag?
ERROR 2025-09-30 11:10:39,153 basehttp "GET /profile/ HTTP/1.1" 500 205796
ERROR 2025-09-30 11:10:39,397 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'endif'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 862, in do_for
    nodelist_loop = parser.parse(
                    ^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 231: 'endif', expected 'empty' or 'endfor'. Did you forget to register or load this tag?
ERROR 2025-09-30 11:10:39,398 basehttp "GET /profile/ HTTP/1.1" 500 205796
INFO 2025-09-30 11:13:16,148 basehttp "GET /profile/ HTTP/1.1" 200 10980
INFO 2025-09-30 11:13:16,653 basehttp "GET /profile/ HTTP/1.1" 200 10980
INFO 2025-09-30 11:13:16,854 basehttp "GET /profile/ HTTP/1.1" 200 10980
INFO 2025-09-30 11:13:17,038 basehttp "GET /profile/ HTTP/1.1" 200 10980
INFO 2025-09-30 11:15:03,483 basehttp "GET /profile/ HTTP/1.1" 200 13119
INFO 2025-09-30 11:15:04,112 basehttp "GET /profile/ HTTP/1.1" 200 13119
INFO 2025-09-30 11:15:04,289 basehttp "GET /profile/ HTTP/1.1" 200 13119
ERROR 2025-09-30 11:18:11,713 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 81, in reverse
    extra, resolver = resolver.namespace_dict[ns]
                      ~~~~~~~~~~~~~~~~~~~~~~~^^^^
KeyError: 'accounts'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 92, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'accounts' is not a registered namespace
ERROR 2025-09-30 11:18:11,713 basehttp "GET /profile/ HTTP/1.1" 500 132488
ERROR 2025-09-30 11:18:12,622 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 81, in reverse
    extra, resolver = resolver.namespace_dict[ns]
                      ~~~~~~~~~~~~~~~~~~~~~~~^^^^
KeyError: 'accounts'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 92, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'accounts' is not a registered namespace
ERROR 2025-09-30 11:18:12,623 basehttp "GET /profile/ HTTP/1.1" 500 132488
ERROR 2025-09-30 11:18:12,823 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 81, in reverse
    extra, resolver = resolver.namespace_dict[ns]
                      ~~~~~~~~~~~~~~~~~~~~~~~^^^^
KeyError: 'accounts'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 92, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'accounts' is not a registered namespace
ERROR 2025-09-30 11:18:12,823 basehttp "GET /profile/ HTTP/1.1" 500 132488
ERROR 2025-09-30 11:18:35,480 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 81, in reverse
    extra, resolver = resolver.namespace_dict[ns]
                      ~~~~~~~~~~~~~~~~~~~~~~~^^^^
KeyError: 'accounts'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 92, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'accounts' is not a registered namespace
ERROR 2025-09-30 11:18:35,481 basehttp "GET /profile/ HTTP/1.1" 500 132326
ERROR 2025-09-30 11:18:53,152 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 81, in reverse
    extra, resolver = resolver.namespace_dict[ns]
                      ~~~~~~~~~~~~~~~~~~~~~~~^^^^
KeyError: 'accounts'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 92, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'accounts' is not a registered namespace
ERROR 2025-09-30 11:18:53,152 basehttp "GET /profile/ HTTP/1.1" 500 132326
ERROR 2025-09-30 11:18:53,688 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 81, in reverse
    extra, resolver = resolver.namespace_dict[ns]
                      ~~~~~~~~~~~~~~~~~~~~~~~^^^^
KeyError: 'accounts'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 92, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'accounts' is not a registered namespace
ERROR 2025-09-30 11:18:53,689 basehttp "GET /profile/ HTTP/1.1" 500 132326
ERROR 2025-09-30 11:18:54,041 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 81, in reverse
    extra, resolver = resolver.namespace_dict[ns]
                      ~~~~~~~~~~~~~~~~~~~~~~~^^^^
KeyError: 'accounts'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 92, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'accounts' is not a registered namespace
ERROR 2025-09-30 11:18:54,042 basehttp "GET /profile/ HTTP/1.1" 500 132326
ERROR 2025-09-30 11:18:54,249 log Internal Server Error: /profile/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 81, in reverse
    extra, resolver = resolver.namespace_dict[ns]
                      ~~~~~~~~~~~~~~~~~~~~~~~^^^^
KeyError: 'accounts'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 215, in user_profile_page
    return render(request, 'accounts/user_informationtemp.html', context = {'user' : user})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 1016, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/urls/base.py", line 92, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'accounts' is not a registered namespace
ERROR 2025-09-30 11:18:54,250 basehttp "GET /profile/ HTTP/1.1" 500 132326
INFO 2025-09-30 11:19:19,166 basehttp "GET /profile/ HTTP/1.1" 200 10704
INFO 2025-09-30 11:19:19,917 basehttp "GET /profile/ HTTP/1.1" 200 10704
INFO 2025-09-30 11:19:20,091 basehttp "GET /profile/ HTTP/1.1" 200 10704
INFO 2025-09-30 11:23:41,035 basehttp "GET /profile/ HTTP/1.1" 200 12203
INFO 2025-09-30 11:23:41,568 basehttp "GET /profile/ HTTP/1.1" 200 12203
INFO 2025-09-30 11:23:41,776 basehttp "GET /profile/ HTTP/1.1" 200 12203
INFO 2025-09-30 11:23:41,936 basehttp "GET /profile/ HTTP/1.1" 200 12203
INFO 2025-09-30 11:25:41,550 basehttp "GET /profile/ HTTP/1.1" 200 12256
INFO 2025-09-30 11:25:42,322 basehttp "GET /profile/ HTTP/1.1" 200 12256
INFO 2025-09-30 11:25:42,555 basehttp "GET /profile/ HTTP/1.1" 200 12256
INFO 2025-09-30 11:32:11,208 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 11:32:11,388 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 11:32:18,424 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 11:32:18,589 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 11:32:23,996 basehttp "GET /profile/ HTTP/1.1" 200 15228
INFO 2025-09-30 11:32:24,503 basehttp "GET /profile/ HTTP/1.1" 200 15228
INFO 2025-09-30 11:32:24,719 basehttp "GET /profile/ HTTP/1.1" 200 15228
INFO 2025-09-30 11:32:24,895 basehttp "GET /profile/ HTTP/1.1" 200 15228
INFO 2025-09-30 11:32:30,951 basehttp "GET /profile/ HTTP/1.1" 200 15228
INFO 2025-09-30 11:32:31,149 basehttp "GET /profile/ HTTP/1.1" 200 15228
INFO 2025-09-30 11:32:31,350 basehttp "GET /profile/ HTTP/1.1" 200 15228
INFO 2025-09-30 11:32:31,525 basehttp "GET /profile/ HTTP/1.1" 200 15228
INFO 2025-09-30 11:32:31,719 basehttp "GET /profile/ HTTP/1.1" 200 15228
INFO 2025-09-30 11:32:31,894 basehttp "GET /profile/ HTTP/1.1" 200 15228
INFO 2025-09-30 11:32:34,587 basehttp "GET /profile/ HTTP/1.1" 200 15228
INFO 2025-09-30 11:34:19,610 basehttp "GET /profile/ HTTP/1.1" 200 15191
INFO 2025-09-30 11:34:20,448 basehttp "GET /profile/ HTTP/1.1" 200 15191
INFO 2025-09-30 11:34:46,211 basehttp "GET /profile/ HTTP/1.1" 200 15123
INFO 2025-09-30 11:34:46,737 basehttp "GET /profile/ HTTP/1.1" 200 15123
INFO 2025-09-30 11:36:03,216 basehttp "POST /profile/ HTTP/1.1" 302 0
INFO 2025-09-30 11:36:03,221 basehttp "GET /profile HTTP/1.1" 301 0
INFO 2025-09-30 11:36:03,225 basehttp "GET /profile/ HTTP/1.1" 200 15769
INFO 2025-09-30 11:36:49,240 basehttp "POST /profile/ HTTP/1.1" 200 15822
INFO 2025-09-30 11:37:26,833 basehttp "POST /profile/ HTTP/1.1" 200 15822
INFO 2025-09-30 11:38:41,441 basehttp "GET /admin/ HTTP/1.1" 302 0
INFO 2025-09-30 11:38:41,450 basehttp "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4177
INFO 2025-09-30 11:38:41,512 basehttp "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810
INFO 2025-09-30 11:38:41,512 basehttp "GET /static/admin/css/login.css HTTP/1.1" 200 951
INFO 2025-09-30 11:38:41,512 basehttp "GET /static/admin/css/responsive.css HTTP/1.1" 200 16565
INFO 2025-09-30 11:38:41,512 basehttp "GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2808
INFO 2025-09-30 11:38:41,513 basehttp "GET /static/admin/css/base.css HTTP/1.1" 200 22120
INFO 2025-09-30 11:38:41,514 basehttp "GET /static/admin/js/theme.js HTTP/1.1" 200 1653
INFO 2025-09-30 11:38:41,517 basehttp "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063
INFO 2025-09-30 11:38:57,854 basehttp "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4366
INFO 2025-09-30 11:39:13,203 basehttp "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4369
INFO 2025-09-30 11:39:52,400 basehttp "POST /profile/ HTTP/1.1" 200 15804
INFO 2025-09-30 11:40:14,633 basehttp "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4355
INFO 2025-09-30 11:40:23,829 basehttp "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4355
INFO 2025-09-30 11:40:37,098 basehttp "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2025-09-30 11:40:37,107 basehttp "GET /admin/ HTTP/1.1" 200 8553
INFO 2025-09-30 11:40:37,155 basehttp "GET /static/admin/css/dashboard.css HTTP/1.1" 200 441
INFO 2025-09-30 11:40:37,200 basehttp "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
INFO 2025-09-30 11:40:37,200 basehttp "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
INFO 2025-09-30 11:40:40,374 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10798
INFO 2025-09-30 11:40:40,382 basehttp "GET /static/admin/css/changelists.css HTTP/1.1" 200 6878
INFO 2025-09-30 11:40:40,386 basehttp "GET /static/admin/js/core.js HTTP/1.1" 200 6208
INFO 2025-09-30 11:40:40,386 basehttp "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 347
INFO 2025-09-30 11:40:40,386 basehttp "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 9777
INFO 2025-09-30 11:40:40,387 basehttp "GET /static/admin/js/actions.js HTTP/1.1" 200 8076
INFO 2025-09-30 11:40:40,388 basehttp "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 285314
INFO 2025-09-30 11:40:40,390 basehttp "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1531
INFO 2025-09-30 11:40:40,390 basehttp "GET /static/admin/js/urlify.js HTTP/1.1" 200 7887
INFO 2025-09-30 11:40:40,391 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 11:40:40,391 basehttp "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 325171
INFO 2025-09-30 11:40:40,434 basehttp "GET /static/admin/js/filters.js HTTP/1.1" 200 978
INFO 2025-09-30 11:40:40,442 basehttp "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
INFO 2025-09-30 11:40:41,574 basehttp "GET /admin/accounts/useraccount/35/change/ HTTP/1.1" 200 21252
INFO 2025-09-30 11:40:41,585 basehttp "GET /static/admin/css/forms.css HTTP/1.1" 200 8525
INFO 2025-09-30 11:40:41,585 basehttp "GET /static/admin/js/calendar.js HTTP/1.1" 200 9141
INFO 2025-09-30 11:40:41,585 basehttp "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 586
INFO 2025-09-30 11:40:41,585 basehttp "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 200 19319
INFO 2025-09-30 11:40:41,588 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 11:40:41,591 basehttp "GET /static/admin/js/change_form.js HTTP/1.1" 200 606
INFO 2025-09-30 11:40:41,626 basehttp "GET /static/admin/css/widgets.css HTTP/1.1" 200 11991
INFO 2025-09-30 11:40:41,690 basehttp "GET /static/admin/img/icon-calendar.svg HTTP/1.1" 200 1086
INFO 2025-09-30 11:40:41,690 basehttp "GET /static/admin/img/icon-clock.svg HTTP/1.1" 200 677
INFO 2025-09-30 11:40:49,779 basehttp "POST /admin/accounts/useraccount/35/change/ HTTP/1.1" 302 0
INFO 2025-09-30 11:40:49,788 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 11023
INFO 2025-09-30 11:40:49,841 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 11:40:49,890 basehttp "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
INFO 2025-09-30 11:40:50,998 basehttp "GET /admin/accounts/useraccount/35/change/ HTTP/1.1" 200 21252
INFO 2025-09-30 11:40:51,009 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 11:40:59,856 basehttp "GET /login/ HTTP/1.1" 200 8226
INFO 2025-09-30 11:41:11,273 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10798
INFO 2025-09-30 11:41:21,579 basehttp "POST /login/ HTTP/1.1" 302 0
INFO 2025-09-30 11:41:21,585 basehttp "GET /home/ HTTP/1.1" 200 2720
INFO 2025-09-30 11:41:23,182 basehttp "GET /profile/ HTTP/1.1" 200 15115
INFO 2025-09-30 11:41:51,019 basehttp "POST /profile/ HTTP/1.1" 302 0
INFO 2025-09-30 11:41:51,023 basehttp "GET /profile HTTP/1.1" 301 0
INFO 2025-09-30 11:41:51,029 basehttp "GET /profile/ HTTP/1.1" 200 15782
INFO 2025-09-30 11:44:01,148 basehttp "POST /profile/ HTTP/1.1" 200 15796
INFO 2025-09-30 11:45:56,496 basehttp "POST /profile/ HTTP/1.1" 200 15856
INFO 2025-09-30 11:46:24,343 basehttp "POST /profile/ HTTP/1.1" 200 15856
INFO 2025-09-30 11:47:14,378 basehttp "POST /profile/ HTTP/1.1" 200 15856
INFO 2025-09-30 11:47:25,479 basehttp "POST /profile/ HTTP/1.1" 200 15856
INFO 2025-09-30 11:49:10,735 basehttp "GET /login/ HTTP/1.1" 200 8226
INFO 2025-09-30 11:49:19,619 basehttp "POST /login/ HTTP/1.1" 302 0
INFO 2025-09-30 11:49:19,623 basehttp "GET /home/ HTTP/1.1" 200 2721
INFO 2025-09-30 11:49:21,353 basehttp "GET /profile/ HTTP/1.1" 200 15155
INFO 2025-09-30 11:49:38,753 basehttp "POST /profile/ HTTP/1.1" 302 0
INFO 2025-09-30 11:49:38,758 basehttp "GET /profile HTTP/1.1" 301 0
INFO 2025-09-30 11:49:38,763 basehttp "GET /profile/ HTTP/1.1" 200 15801
INFO 2025-09-30 11:50:31,930 basehttp "POST /profile/ HTTP/1.1" 200 15899
INFO 2025-09-30 11:50:55,633 basehttp "POST /profile/ HTTP/1.1" 200 15911
INFO 2025-09-30 11:52:47,367 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 11:52:47,534 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 11:52:51,981 basehttp "POST /profile/ HTTP/1.1" 200 15858
INFO 2025-09-30 11:53:01,377 basehttp "POST /profile/ HTTP/1.1" 200 15858
INFO 2025-09-30 11:53:27,687 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 11:53:27,861 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 11:53:30,953 basehttp "GET /profile/ HTTP/1.1" 200 15164
INFO 2025-09-30 11:53:41,993 basehttp "POST /profile/ HTTP/1.1" 200 15858
INFO 2025-09-30 11:56:29,331 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 11:56:29,507 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 11:58:11,777 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 11:58:11,972 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 11:59:36,206 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 11:59:36,380 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:00:24,539 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:00:24,710 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:00:32,783 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:00:32,960 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:00:49,049 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:00:49,220 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:00:54,282 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:00:54,449 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:00:55,501 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:00:55,684 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:00:57,391 basehttp "POST /profile/ HTTP/1.1" 200 15866
INFO 2025-09-30 12:00:59,832 basehttp "GET /profile/ HTTP/1.1" 200 15164
INFO 2025-09-30 12:01:03,329 basehttp "POST /profile/ HTTP/1.1" 200 15866
INFO 2025-09-30 12:01:17,677 basehttp "POST /profile/ HTTP/1.1" 200 15866
INFO 2025-09-30 12:02:07,398 basehttp "GET /login/ HTTP/1.1" 200 8226
INFO 2025-09-30 12:02:28,727 basehttp "POST /login/ HTTP/1.1" 302 0
INFO 2025-09-30 12:02:28,732 basehttp "GET /home/ HTTP/1.1" 200 2721
INFO 2025-09-30 12:02:30,902 basehttp "GET /profile/ HTTP/1.1" 200 15164
INFO 2025-09-30 12:02:38,076 basehttp "POST /profile/ HTTP/1.1" 200 15866
INFO 2025-09-30 12:03:46,128 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:03:46,309 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:03:48,028 basehttp "POST /profile/ HTTP/1.1" 302 0
INFO 2025-09-30 12:03:48,033 basehttp "GET /profile HTTP/1.1" 301 0
INFO 2025-09-30 12:03:48,038 basehttp "GET /profile/ HTTP/1.1" 200 16526
INFO 2025-09-30 12:03:51,621 basehttp "GET /profile/ HTTP/1.1" 200 15155
INFO 2025-09-30 12:04:14,558 basehttp "POST /profile/ HTTP/1.1" 302 0
INFO 2025-09-30 12:04:14,563 basehttp "GET /profile HTTP/1.1" 301 0
INFO 2025-09-30 12:04:14,569 basehttp "GET /profile/ HTTP/1.1" 200 15809
INFO 2025-09-30 12:04:19,445 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:04:19,639 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:04:42,375 basehttp "POST /profile/ HTTP/1.1" 302 0
INFO 2025-09-30 12:04:42,381 basehttp "GET /profile HTTP/1.1" 301 0
INFO 2025-09-30 12:04:42,386 basehttp "GET /profile/ HTTP/1.1" 200 15793
INFO 2025-09-30 12:05:14,703 basehttp "GET /login/ HTTP/1.1" 200 8226
INFO 2025-09-30 12:05:27,564 basehttp "POST /login/ HTTP/1.1" 302 0
INFO 2025-09-30 12:05:27,569 basehttp "GET /home/ HTTP/1.1" 200 2719
INFO 2025-09-30 12:05:29,937 basehttp "GET /profile/ HTTP/1.1" 200 15123
INFO 2025-09-30 12:05:37,350 basehttp "GET /login/ HTTP/1.1" 200 8226
INFO 2025-09-30 12:05:45,990 basehttp "POST /login/ HTTP/1.1" 200 8477
INFO 2025-09-30 12:06:09,439 basehttp "POST /login/ HTTP/1.1" 302 0
INFO 2025-09-30 12:06:09,443 basehttp "GET /home/ HTTP/1.1" 200 2720
INFO 2025-09-30 12:06:11,167 basehttp "GET /profile/ HTTP/1.1" 200 15145
INFO 2025-09-30 12:06:27,631 basehttp "POST /profile/ HTTP/1.1" 302 0
INFO 2025-09-30 12:06:27,635 basehttp "GET /profile HTTP/1.1" 301 0
INFO 2025-09-30 12:06:27,639 basehttp "GET /profile/ HTTP/1.1" 200 16484
INFO 2025-09-30 12:07:12,153 basehttp "POST /profile/ HTTP/1.1" 302 0
INFO 2025-09-30 12:07:12,157 basehttp "GET /profile HTTP/1.1" 301 0
INFO 2025-09-30 12:07:12,161 basehttp "GET /profile/ HTTP/1.1" 200 15753
INFO 2025-09-30 12:07:33,024 basehttp "GET /admin/accounts/useraccount/34/change/ HTTP/1.1" 200 21330
INFO 2025-09-30 12:07:33,038 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:08:29,110 basehttp "GET /login/ HTTP/1.1" 200 8226
INFO 2025-09-30 12:09:03,570 basehttp "POST /login/ HTTP/1.1" 302 0
INFO 2025-09-30 12:09:03,575 basehttp "GET /home/ HTTP/1.1" 200 2719
INFO 2025-09-30 12:09:05,161 basehttp "GET /profile/ HTTP/1.1" 200 15123
INFO 2025-09-30 12:09:13,437 basehttp "POST /profile/ HTTP/1.1" 200 15831
INFO 2025-09-30 12:09:29,507 basehttp "GET /admin/ HTTP/1.1" 200 8927
INFO 2025-09-30 12:09:31,338 basehttp "GET /admin/accounts/useraccount/add/ HTTP/1.1" 200 20816
INFO 2025-09-30 12:09:31,355 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:09:33,711 basehttp "GET /admin/ HTTP/1.1" 200 8927
INFO 2025-09-30 12:09:34,582 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10798
INFO 2025-09-30 12:09:34,598 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:09:41,874 basehttp "GET /admin/accounts/useraccount/30/change/ HTTP/1.1" 200 21355
INFO 2025-09-30 12:09:41,884 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:09:44,696 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10798
INFO 2025-09-30 12:09:45,800 basehttp "GET /admin/accounts/useraccount/30/change/ HTTP/1.1" 200 21355
INFO 2025-09-30 12:09:45,812 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:09:48,573 basehttp "GET /admin/accounts/useraccount/30/delete/ HTTP/1.1" 200 8554
INFO 2025-09-30 12:09:48,598 basehttp "GET /static/admin/js/cancel.js HTTP/1.1" 200 884
INFO 2025-09-30 12:09:49,675 basehttp "POST /admin/accounts/useraccount/30/delete/ HTTP/1.1" 302 0
INFO 2025-09-30 12:09:49,678 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 302 0
INFO 2025-09-30 12:09:49,725 basehttp "GET /admin/login/?next=/admin/accounts/useraccount/ HTTP/1.1" 200 4391
INFO 2025-09-30 12:10:00,873 basehttp "POST /admin/login/?next=/admin/accounts/useraccount/ HTTP/1.1" 200 4397
INFO 2025-09-30 12:11:05,369 basehttp "POST /admin/login/?next=/admin/accounts/useraccount/ HTTP/1.1" 302 0
INFO 2025-09-30 12:11:05,378 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10810
INFO 2025-09-30 12:11:05,430 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:11:07,119 basehttp "GET /admin/accounts/useraccount/36/change/ HTTP/1.1" 200 21314
INFO 2025-09-30 12:11:07,131 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:11:10,338 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10810
INFO 2025-09-30 12:11:16,809 basehttp "GET /admin/accounts/useraccount/35/change/ HTTP/1.1" 200 21300
INFO 2025-09-30 12:11:16,822 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:11:21,418 basehttp "GET /admin/accounts/useraccount/35/delete/ HTTP/1.1" 200 7941
INFO 2025-09-30 12:11:22,558 basehttp "POST /admin/accounts/useraccount/35/delete/ HTTP/1.1" 302 0
INFO 2025-09-30 12:11:22,566 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10674
INFO 2025-09-30 12:11:22,622 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:11:23,465 basehttp "GET /admin/accounts/useraccount/36/change/ HTTP/1.1" 200 21314
INFO 2025-09-30 12:11:23,476 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:11:31,573 basehttp "POST /admin/accounts/useraccount/36/change/ HTTP/1.1" 302 0
INFO 2025-09-30 12:11:31,581 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10724
INFO 2025-09-30 12:11:31,634 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:11:38,569 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10499
INFO 2025-09-30 12:11:38,579 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:11:38,984 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10499
INFO 2025-09-30 12:11:38,998 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:11:41,014 basehttp "POST /admin/logout/ HTTP/1.1" 200 3572
INFO 2025-09-30 12:11:42,394 basehttp "GET /admin/ HTTP/1.1" 302 0
INFO 2025-09-30 12:11:42,398 basehttp "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4177
INFO 2025-09-30 12:11:53,581 basehttp "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2025-09-30 12:11:53,588 basehttp "GET /admin/ HTTP/1.1" 200 7380
INFO 2025-09-30 12:11:53,640 basehttp "GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392
INFO 2025-09-30 12:12:00,082 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10499
INFO 2025-09-30 12:12:00,095 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:12:03,223 basehttp "GET /admin/accounts/useraccount/33/change/ HTTP/1.1" 200 21315
INFO 2025-09-30 12:12:03,234 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:12:25,036 basehttp "POST /profile/ HTTP/1.1" 302 0
WARNING 2025-09-30 12:12:25,040 log Not Found: /accounts/login/
WARNING 2025-09-30 12:12:25,040 basehttp "GET /accounts/login/?next=/profile/ HTTP/1.1" 404 4086
WARNING 2025-09-30 12:12:33,348 log Not Found: /accounts/login/
WARNING 2025-09-30 12:12:33,348 basehttp "GET /accounts/login/?next=/profile/ HTTP/1.1" 404 4086
INFO 2025-09-30 12:12:35,668 basehttp "POST /profile/ HTTP/1.1" 302 0
WARNING 2025-09-30 12:12:35,671 log Not Found: /accounts/login/
WARNING 2025-09-30 12:12:35,671 basehttp "GET /accounts/login/?next=/profile/ HTTP/1.1" 404 4086
INFO 2025-09-30 12:12:36,963 basehttp "GET /profile/ HTTP/1.1" 302 0
INFO 2025-09-30 12:12:44,514 basehttp "GET /login/ HTTP/1.1" 200 8226
INFO 2025-09-30 12:12:54,482 basehttp "POST /login/ HTTP/1.1" 302 0
INFO 2025-09-30 12:12:54,486 basehttp "GET /home/ HTTP/1.1" 200 2713
INFO 2025-09-30 12:12:56,520 basehttp "GET /profile/ HTTP/1.1" 200 15049
INFO 2025-09-30 12:13:14,191 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10499
INFO 2025-09-30 12:13:15,601 basehttp "GET /admin/accounts/useraccount/36/change/ HTTP/1.1" 200 21308
INFO 2025-09-30 12:13:15,613 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:14:21,569 basehttp "GET /admin/accounts/useraccount/36/delete/ HTTP/1.1" 200 8133
INFO 2025-09-30 12:14:23,101 basehttp "POST /admin/accounts/useraccount/36/delete/ HTTP/1.1" 302 0
INFO 2025-09-30 12:14:23,104 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 302 0
INFO 2025-09-30 12:14:23,149 basehttp "GET /admin/login/?next=/admin/accounts/useraccount/ HTTP/1.1" 200 4391
INFO 2025-09-30 12:15:39,900 basehttp "POST /admin/login/?next=/admin/accounts/useraccount/ HTTP/1.1" 302 0
INFO 2025-09-30 12:15:39,910 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10490
INFO 2025-09-30 12:15:39,961 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:15:42,153 basehttp "GET /admin/accounts/useraccount/37/change/ HTTP/1.1" 200 21352
INFO 2025-09-30 12:15:42,167 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:15:44,406 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10490
INFO 2025-09-30 12:15:54,365 basehttp "GET /profile/ HTTP/1.1" 302 0
WARNING 2025-09-30 12:15:54,372 log Not Found: /accounts/login/
WARNING 2025-09-30 12:15:54,372 basehttp "GET /accounts/login/?next=/profile/ HTTP/1.1" 404 4086
WARNING 2025-09-30 12:15:54,812 log Not Found: /accounts/login/
WARNING 2025-09-30 12:15:54,812 basehttp "GET /accounts/login/?next=/profile/ HTTP/1.1" 404 4086
WARNING 2025-09-30 12:15:55,044 log Not Found: /accounts/login/
WARNING 2025-09-30 12:15:55,044 basehttp "GET /accounts/login/?next=/profile/ HTTP/1.1" 404 4086
WARNING 2025-09-30 12:15:55,252 log Not Found: /accounts/login/
WARNING 2025-09-30 12:15:55,252 basehttp "GET /accounts/login/?next=/profile/ HTTP/1.1" 404 4086
INFO 2025-09-30 12:16:00,243 basehttp "GET /login/ HTTP/1.1" 200 8226
INFO 2025-09-30 12:16:10,663 basehttp "POST /login/ HTTP/1.1" 200 8478
INFO 2025-09-30 12:16:36,776 basehttp "GET /admin/accounts/useraccount/37/change/ HTTP/1.1" 200 21352
INFO 2025-09-30 12:16:36,790 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:16:47,664 basehttp "POST /admin/accounts/useraccount/37/change/ HTTP/1.1" 302 0
INFO 2025-09-30 12:16:47,672 basehttp "GET /admin/accounts/useraccount/ HTTP/1.1" 200 10715
INFO 2025-09-30 12:16:47,724 basehttp "GET /admin/jsi18n/ HTTP/1.1" 200 3342
INFO 2025-09-30 12:17:35,517 basehttp "POST /login/ HTTP/1.1" 302 0
INFO 2025-09-30 12:17:35,521 basehttp "GET /home/ HTTP/1.1" 200 2719
INFO 2025-09-30 12:17:37,448 basehttp "GET /profile/ HTTP/1.1" 200 15123
INFO 2025-09-30 12:17:44,698 basehttp "POST /profile/ HTTP/1.1" 302 0
INFO 2025-09-30 12:17:44,702 basehttp "GET /profile HTTP/1.1" 301 0
INFO 2025-09-30 12:17:44,706 basehttp "GET /profile/ HTTP/1.1" 200 15761
INFO 2025-09-30 12:25:23,585 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:25:23,774 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:27:20,085 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:27:20,259 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:29:25,583 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/urls.py changed, reloading.
INFO 2025-09-30 12:29:25,754 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:29:39,838 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:29:40,012 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:31:05,241 basehttp "GET /profile/ HTTP/1.1" 200 15130
INFO 2025-09-30 12:31:05,523 basehttp "GET /profile/ HTTP/1.1" 200 15130
INFO 2025-09-30 12:31:05,734 basehttp "GET /profile/ HTTP/1.1" 200 15130
INFO 2025-09-30 12:31:05,893 basehttp "GET /profile/ HTTP/1.1" 200 15130
INFO 2025-09-30 12:31:11,094 basehttp "GET /profile/ HTTP/1.1" 200 15130
WARNING 2025-09-30 12:31:13,558 log Not Found: /logout/
WARNING 2025-09-30 12:31:13,558 basehttp "GET /logout/ HTTP/1.1" 404 4301
WARNING 2025-09-30 12:32:12,742 log Not Found: /logout/
WARNING 2025-09-30 12:32:12,742 basehttp "GET /logout/ HTTP/1.1" 404 4301
INFO 2025-09-30 12:32:28,471 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/urls.py changed, reloading.
INFO 2025-09-30 12:32:28,649 autoreload Watching for file changes with StatReloader
ERROR 2025-09-30 12:32:32,417 log Internal Server Error: /logout/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 302, in logout
    logout(request)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 302, in logout
    logout(request)
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 302, in logout
    logout(request)
  [Previous line repeated 959 more times]
RecursionError: maximum recursion depth exceeded
ERROR 2025-09-30 12:32:32,426 basehttp "GET /logout/ HTTP/1.1" 500 3127687
INFO 2025-09-30 12:33:47,891 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:33:48,081 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:34:35,244 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:34:35,452 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:34:44,559 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/urls.py changed, reloading.
INFO 2025-09-30 12:34:44,741 autoreload Watching for file changes with StatReloader
ERROR 2025-09-30 12:34:47,779 log Internal Server Error: /logout/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 302, in logout_user
    djanologout(request)
    ^^^^^^^^^^^
NameError: name 'djanologout' is not defined
ERROR 2025-09-30 12:34:47,779 basehttp "GET /logout/ HTTP/1.1" 500 70798
INFO 2025-09-30 12:34:51,247 basehttp "GET /profile/ HTTP/1.1" 200 15130
INFO 2025-09-30 12:34:51,510 basehttp "GET /profile/ HTTP/1.1" 200 15130
INFO 2025-09-30 12:34:51,726 basehttp "GET /profile/ HTTP/1.1" 200 15130
INFO 2025-09-30 12:34:51,932 basehttp "GET /profile/ HTTP/1.1" 200 15130
INFO 2025-09-30 12:34:52,133 basehttp "GET /profile/ HTTP/1.1" 200 15130
ERROR 2025-09-30 12:34:53,165 log Internal Server Error: /logout/
Traceback (most recent call last):
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py", line 302, in logout_user
    djanologout(request)
    ^^^^^^^^^^^
NameError: name 'djanologout' is not defined
ERROR 2025-09-30 12:34:53,165 basehttp "GET /logout/ HTTP/1.1" 500 70798
INFO 2025-09-30 12:35:04,881 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:35:05,048 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:35:07,041 basehttp "GET /logout/ HTTP/1.1" 302 0
INFO 2025-09-30 12:35:07,045 basehttp "GET /login/ HTTP/1.1" 200 8451
INFO 2025-09-30 12:35:07,858 basehttp "GET /login/ HTTP/1.1" 200 8226
INFO 2025-09-30 12:35:08,050 basehttp "GET /login/ HTTP/1.1" 200 8226
INFO 2025-09-30 12:35:08,636 basehttp "GET /login/ HTTP/1.1" 200 8226
INFO 2025-09-30 12:35:08,859 basehttp "GET /login/ HTTP/1.1" 200 8226
INFO 2025-09-30 12:35:15,296 basehttp "GET /home/ HTTP/1.1" 302 0
WARNING 2025-09-30 12:35:15,300 log Not Found: /accounts/login/
WARNING 2025-09-30 12:35:15,301 basehttp "GET /accounts/login/?next=/home/ HTTP/1.1" 404 4338
INFO 2025-09-30 12:35:34,781 basehttp "POST /login/ HTTP/1.1" 302 0
INFO 2025-09-30 12:35:34,785 basehttp "GET /home/ HTTP/1.1" 200 2726
INFO 2025-09-30 12:35:37,072 basehttp "GET /logout/ HTTP/1.1" 302 0
INFO 2025-09-30 12:35:37,075 basehttp "GET /login/ HTTP/1.1" 200 8451
INFO 2025-09-30 12:36:49,364 autoreload /home/pranav_vasankar/IUCAA_tasks/authentication_system_task/authentication_system/accounts/views.py changed, reloading.
INFO 2025-09-30 12:36:49,541 autoreload Watching for file changes with StatReloader
INFO 2025-09-30 12:37:10,503 basehttp "GET /forgot_password/ HTTP/1.1" 200 12189
INFO 2025-09-30 12:37:49,333 basehttp "GET /forgot_password/ HTTP/1.1" 200 12189
